### PR TITLE
NAS-101056 Fix text that was referring to USB devices other than for install only.

### PR DESCRIPTION
**Pull request by a-stjohn**
cherry-picked from PR #844

### CODE REVIEW

File: userguide/sharing.rst [hunk 1]
```diff
@@ -1527,7 +1527,7 @@ connection requests.
 initiator negotiates with a target to establish connectivity to a LUN.
 The result is an iSCSI connection that emulates a connection to a SCSI
 hard disk. Initiators treat iSCSI LUNs as if they were a raw SCSI or
-SATA hard drive. Rather than mounting remote directories, initiators
+SSD. Rather than mounting remote directories, initiators
```
Comment (wblock) [inline userguide/sharing.rst]:
```suggestion
SATA hard drive. Rather than mounting remote directories, initiators
```

File: userguide/install.rst [hunk 2]
```diff
@@ -66,33 +66,32 @@ in the :file:`sha256.txt` file.  Checksum values that do not match
...
-.. index:: Burn ISO, ISO, USB Stick
+.. index:: Burn ISO, ISO
 .. _Preparing the Media:
 
 Preparing the Media
 -------------------
 
-The %brand% installer can run from either a CD or a USB memory
-stick.
+The %brand% installer can run from either a CD or a |usb-stick|.
```
Comment (wblock) [inline userguide/install.rst]:
```suggestion
The %brand% installer can run from |usb-stick| or a CD.
```

File: userguide/install.rst [hunk 3]
```diff
@@ -38,8 +38,8 @@ from `<https://download.freenas.org/11.2/STABLE/latest/>`__.
 .. note:: %brand% requires 64-bit hardware.
 
 The download page contains an *.iso* file. This is a bootable
-installer that can be written to either a CD or USB flash as described
-in :ref:`Preparing the Media`.
+installer that can be written to either a CD or |usb-stick| as
```
Comment (wblock) [inline userguide/install.rst]:
```suggestion
installer that can be written to a |usb-stick| or CD
```

File: userguide/install.rst [hunk 4]
```diff
@@ -38,8 +38,8 @@ from `<https://download.freenas.org/11.2/STABLE/latest/>`__.
 .. note:: %brand% requires 64-bit hardware.
 
 The download page contains an *.iso* file. This is a bootable
-installer that can be written to either a CD or USB flash as described
-in :ref:`Preparing the Media`.
+installer that can be written to a |usb-stick| or CD
```
Comment (wblock) [inline userguide/install.rst]:
```suggestion
installer that can be written to a |usb-stick| or CD as
```

File: userguide/install.rst [hunk 5]
```diff
@@ -66,33 +66,32 @@ in the :file:`sha256.txt` file.  Checksum values that do not match
...
-stick.
+The %brand% installer can run from |usb-stick| or a CD.
 
 A CD burning utility is needed to write the :file:`.iso` file to a
 CD.
 
-The :file:`.iso` file can also be written to a USB memory stick. The
-method used to write the file depends on the operating system.
-Examples for several common operating systems are shown below.
+The :file:`.iso` file can also be written to a |usb-stick|. The
```
Comment (wblock) [inline userguide/install.rst]:
```suggestion
The :file:`.iso` file can be written directly to a |usb-stick|. The
```

File: userguide/intro.rst [hunk 6]
```diff
@@ -675,19 +675,19 @@ The Operating System Device
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The %brand% operating system is installed to at least one device that
-is separate from the storage disks. The device can be a SSD, USB
-memory stick, or DOM (Disk on Module). Installation to a hard drive is
+is separate from the storage disks. The device can be a SSD or
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
is separate from the storage disks. The device can be a SSD, a small hard drive, or a
```

File: userguide/intro.rst [hunk 7]
```diff
@@ -675,19 +675,19 @@ The Operating System Device
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The %brand% operating system is installed to at least one device that
-is separate from the storage disks. The device can be a SSD, USB
-memory stick, or DOM (Disk on Module). Installation to a hard drive is
+is separate from the storage disks. The device can be a SSD, a small hard drive, or a
+|usb-stick|. Installation to a hard drive is
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
|usb-stick|.
```

File: userguide/intro.rst [hunk 8]
```diff
@@ -675,19 +675,19 @@ The Operating System Device
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The %brand% operating system is installed to at least one device that
-is separate from the storage disks. The device can be a SSD, USB
-memory stick, or DOM (Disk on Module). Installation to a hard drive is
+is separate from the storage disks. The device can be a SSD, a small hard drive, or a
+|usb-stick|.
 discouraged as that drive is then not available for data storage.
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
```

File: userguide/system.rst [hunk 9]
```diff
@@ -455,63 +454,62 @@ There are also other options available.
...
-Mirroring the Boot Device
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Mirroring the |OS-Device|
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If the system is currently booting from a device, another device can
-be added to create a mirrored boot device. If one
-device in a mirror fails, the remaining device can still be used to boot
-the system.
+If the system is currently booting from a device, another device
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
If the system is currently booting from a single |os-device|, another device
```

File: userguide/system.rst [hunk 10]
```diff
@@ -536,7 +534,7 @@ Click :guilabel:`SAVE` to attach the new disk to the mirror.
 
 .. figure:: images/system-boot-attach.png
 
-   Mirroring a Boot Device
+   Mirroring a |OS-Device|
```
Comment (wblock) [inline userguide/system.rst]:
```suggestion
   Mirroring an |OS-Device|
```

File: userguide/system.rst [hunk 11]
```diff
@@ -434,17 +434,16 @@ There are also other options available.
   in the :guilabel:`Active` column. Only alphanumeric characters,
   underscores, and dashes are allowed in the name.
 
-* **Scrub:** :guilabel:`Scrub Boot Pool` is used to perform a
```
Comment (a-stjohn) [inline userguide/system.rst]: The text in red is what should be present in this branch. Revert this change, with the addition of changing boot device to |os-device|.

File: userguide/system.rst [hunk 12]
```diff
@@ -434,17 +434,16 @@ There are also other options available.
...
-  last scrub are also listed in this screen. The condition of the boot
-  device should be listed as *HEALTHY*.
+* **Scrub Boot:** can be used to perform a manual scrub of the boot
+  devices. By default, the |os-device| is scrubbed every 7 days. To
+  change the default interval, change the number in the
+  :guilabel:`Automatic scrub interval (in days)` field. The date and
+  results of the last scrub are also listed in this screen. The
+  condition of the |os-device| should be listed as *HEALTHY*.
 
-* **Status:** click :guilabel:`Boot Pool Status` to see the status of
```
Comment (a-stjohn) [inline userguide/system.rst]: The text in red is what should be present in this branch. Revert this change, with the addition of changing boot device to |os-device|.
